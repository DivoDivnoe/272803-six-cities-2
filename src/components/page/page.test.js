import React from 'react';
import renderer from 'react-test-renderer';
import Page from './page.jsx';

jest.mock(`../header/header.jsx`, () => jest.fn().mockReturnValue(null));

describe(`Page component`, () => {
  it(`is rendered correctly`, () => {
    const tree = renderer
      .create(<Page/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
