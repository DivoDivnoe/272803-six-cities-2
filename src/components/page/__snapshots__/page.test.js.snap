// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page component is rendered correctly 1`] = `
<div
  className="page "
/>
`;
